Fika.Core/Coop/Players/ObservedCoopPlayer.cs

```diff
@@ -1021,7 +1021,7 @@ public override void DoObservedVault(ref VaultPacket packet)
 
 		private IEnumerator DestroyNetworkedComponents()
 		{
-			yield return new WaitForSeconds(2);
+			yield return new WaitForSeconds(10);
```

Comment:
```suggestion
			yield return new WaitForSeconds(2);
```
Revert all this and remove the code block below that removes the `Speaker`, it's old code that prevented a bug that should no longer exist.